tests/Support/DatabaseMigrations.php

```diff
@@ -0,0 +1,16 @@
+<?php
+trait DatabaseMigrations
+{
+    /**
+     * @before
+     */
+    public function runDatabaseMigrations()
+    {
+        $this->artisan('migrate');
+        $this->artisan('db:seed');
+
+        $this->beforeApplicationDestroyed(function () {
+            $this->artisan('migrate:rollback');
```

Comment:
its optional trait. let say if in a test i want to do this, then i import this file, if i do not want this behaviour, i do not import. but default, this behaviour does nothing until you import at top of a php file.